distribution/docs/src/main/jdocs/content/_configuring/configuring-managed-service-factory-bundles-contents.adoc

```diff
@@ -31,97 +31,143 @@ The following table shows the format to use for each property type supported.
-|`sizes=D("0x3f8fbe77","0x400ef9db","0x4056147b")`
+|String
+a|name = "value"
+a|`name = "John"`
+
+|Boolean
+a|name = B"true\|false" +
+name = b"true\|false"
+a|`authorized = B"true"`
+
+|Integer
+a|name = I"value" +
+name = i"value"
+a|`timeout = I"10"`
+
+|Long
+a|name = L"value" +
+name = l"value"
+a|`diameter = L"100"`
+
+|Float
+a|name = F"value" +
+name = f"value"
+a|`cost = F"1065353216"`
+
+|Double
+a|name = D"value" +
+name = d"value"
+a|`latitude = D"4636745974857667812"`
+
+|List of Strings
+a|name = ["value1","value2",…​]
+a|```
+complexStringArray = [ \
+  "{\"url\"\ \"http://test.sample.com\"\ \"layers\"\ [\"0\"]\ \"VERSION\"\ \"1.1\|1.2\"\ \"image/png\"}\ \"beta\"\ 1}", \
+  "{\"url\"\ \"http://test.sample.com"\ 0.5}","/solr\=SAML\|PKI\|basic", \
+  "/security-config\=SAML\|basic", \
+  ]
+```
+
+|List of Booleans
+a|name = B["true\|false","true\|false",…​] +
+name = b["true\|false","true\|false",…​]
+a|```
+authorizedList = B[ \
+  "true", \
+  "false", \
+  ]
+```
+
+|List of Integers
+a|name = I["value1","value2",…​] +
+name = i["value1","value2",…​]
+a|```
+sizes = I[ \
+  "10", \
+  "20", \
+  "30", \
+  ]
+```
+
+|List of Longs
+a|name = L["value1","value2",…​] +
+name = l["value1","value2",…​]
+a|```
+sizes = L[ \
+  "100", \
+  "200", \
+  "300", \
+  ]
+```
+
+|List of Floats
+a|name = F["value1","value2",…​] +
+name = f["value1","value2",…​]
+a|```
+sizes = F[ \
+  "1066192077", \
+  "1074580685", \
+  "1079194419", \
+  ]
+```
+
+|List of Doubles
+a|name = D["value1","value2",…​] +
+name = d["value1","value2",…​]
+a|```
+sizes = D[ \
+  "4607736361554183979", \
+  "4612212939583790252", \
+  "4614714689176794563", \
+  ]
+```
 
 |===
 
 [NOTE]
 ====
-* Lists and Vectors of values can be prefixed with any of the supported types (`B`, `I`, `L`, `F` or `D`).
-* To prevent any configuration issues, the `=` signs used in values should be escaped using a backslash (`\`).
+* Lists of values can be prefixed with any of the supported types (`B`, `b`, `I`, `i`, `L`, `l`, `F`, `f`, `D`, or `d`).
+* To prevent any configuration issues, the `=` signs used in values must be escaped using a backslash (`\`).
 * Boolean values will default to `false` if any value other than `true` is provided.
-* Escape character in values must be used for double quotes (`"`) and spaces, but cannot be used with `{` `}` or `[` `]` pairings.
-* Floats and Longs must be written in the IEEE 754 format. For example, the example for the Floats and Doubles in the table above correspond to 10.50 and 45.0234, respectively.
+* Double quotes (`"`) and spaces within values must be escaped using a backslash (`\`), but cannot be used with `{` `}` or `[` `]` pairings.
+* When properties are split over multiple lines for readability, end of lines must be specified with a backslash (`\`). See the examples for lists in the table.
+* Float values must be representated in the IEEE 754 floating-point "single format" bit layout, preserving Not-a-Number (NaN) values.
```

Comment:
❓Should we mention that the long integer number must be an **integer** representation of the binary number used by the IEEE 754 standard to represent floats and doubles?
❓@ricklarsen, is there a way to have those explanations as table footnotes and refer to them from the cells in the table? Do you think it'd be better than what we have now?

Reply:
I added links to the java documentation to provide some more details about the conversions.